artifactory_cleanup/artifactorycleanup.py

```diff
@@ -69,6 +69,9 @@ def _destroy_or_verbose(self):
             print("Verbose MODE")
 
     def main(self):
+        if self._artifactory_server.endswith("/"):
+            # remove trailing slash
+            self._artifactory_server = self._artifactory_server.rstrip("/")
```

Comment:
You can always use multiline suggestions nowadays 😄 
```suggestion
        # remove trailing slash
        self._artifactory_server = self._artifactory_server.rstrip("/")
```